vagrant.md

```diff
@@ -63,6 +63,13 @@ To destroy all VMs
 vagrant(.exe) destroy
 ```
 
+In case vagrant needs to be set using a private network due to issues in getting IPs in the public network the option **--network-type=private_network** should be used.
+```
+vagrant.exe --network-type=private_network up
+vagrant.exe --network-type=private_network reload
```

Comment:
Replace `vagrant.exe` by `vagrant`, `vagrant` is cross-platform compatible.

Why reload? The cluster should be not exist or halted, after running `vagrant up` a reload should not be needed.